### ISSUE TITLE
Sipeed is not a SoC vendor

### ISSUE THREAD

**Issue by Dr-Noob** [Dr-Noob/cpufetch#219]
As pointed out by @Icenowy [here](https://github.com/Dr-Noob/cpufetch/commit/a43b70054081d074ccd6f692aadfb46406fa6f22#r137848994) (thanks!), the SoC vendor is T-Head, Sipeed is only the SBC vendor. 

Actually, the SoC in #200 seems to be TH1520 (T-Head 1520?) according to [this page](https://sipeed.com/licheepi4a).

**Comment by ThomasKaiser** [comment]
> Actually, the SoC in https://github.com/Dr-Noob/cpufetch/issues/200 seems to be TH1520

Yep, [TH1520](https://github.com/sipeed/sipeed_wiki/blob/main/docs/hardware/en/lichee/th1520/lpi4a/1_intro.md): quad-core Xuantie C910 (the cores also developed by T-Head)
